Refactor Homepage UI (#1390)
wpilibsuite/frc-docs: source/index.rst

      :classes: btn-outline-primary btn-block

   ---

   Advanced Programming

   ^^^

   Documentation that is suited toward veteran teams. This includes content such as Path Planning and Kinematics.

   +++

   .. link-button:: stubs/advanced-programming-stub
      :type: ref
      :text: View articles
      :classes: btn-outline-primary btn-block

   ---

   Example Projects

   ^^^

   This section showcases the available example projects that teams can reference in VS Code.

   +++

   .. link-button:: docs/software/examples-tutorials/wpilib-examples
      :type: ref
      :text: View articles
      :classes: btn-outline-primary btn-block

   ---

   Hardware

   ^^^

   Hardware tutorials and content available for teams.

   +++

   .. link-button:: stubs/hardware-stub
      :type: ref
      :text: View articles
      :classes: btn-outline-primary btn-block

   ---

   Romi Robot

   ^^^

   The Romi Robot is a low-cost Raspberry Pi based platform for practicing WPILib programming.

   +++

   .. link-button:: docs/romi-robot/index
      :type: ref
      :text: View articles
      :classes: btn-outline-primary btn-block

   ---

   Contributing

   ^^^

>    Interested in contributing to WPILib? This section will get you started.

Reviewer: This also covers contributing to documentation

FIRST Robotics Competition Control System

> ===========================================

Reviewer 2: remove last 2 =

> Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation! This site will contain everything you need to know for building and programming a competition robot!

Reviewer 2: ```suggestion
Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation! This site contains everything you need to know for programming a competition robot!
```

"will contain" is future tense.  I like using present here.  As is sounds like things are incomplete.

> Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation! This site will contain everything you need to know for building and programming a competition robot!

Reviewer 2: not sure it contains everything needed for "building" a robot but programming we do pretty good.  So maybe cut building?

Author: We do have a build/assembly guide.

Reviewer: The subtlety being that the guide is only for assembling the kitbot, which isn't a full competition robot.

> Community translations can be found for a variety of languages in the bottom-left menu.

Reviewer 2: ```suggestion
Community translations can be found in a variety of languages in the bottom-left menu.
```

.. panels::
   :column: col-sm-6 d-flex pb-3
   :footer: bg-light

   ---
   :header: bg-success font-weight-bold text-white

   Returning Teams

   ^^^

>    If you are a returning team, please check out the overview of changes from 2020 to 2021.

Reviewer: This may undersell what is needed for a returning team

   +++

   .. link-button:: /docs/yearly-overview/yearly-changelog
      :type: ref
      :text: Go to Changelog
      :classes: btn-outline-primary btn-block

   ---
   :header: bg-info font-weight-bold text-white

   New Teams

   ^^^

>    Please read through the Zero-to-Robot tutorial! This will guide you through preparation, building and programming your robot!

Reviewer: ```suggestion
   Please read through the Zero-to-Robot tutorial! This will guide you through preparation, building and programming a basic robot!
```

   +++

   .. link-button:: /docs/zero-to-robot/introduction
      :type: ref
      :text: Go to Zero-to-Robot
      :classes: btn-outline-primary btn-block

.. raw:: html

   <hr>

.. panels::
   :column: col-sm-6 d-flex pb-3
   :footer: bg-light

   ---
   :header: font-weight-bold text-dark

   Hardware Overview

   ^^^

>    An overview of the hardware components available for teams.

Reviewer 2: ```suggestion
   An overview of the hardware components available to teams.
```

   +++

   .. link-button:: /docs/controls-overviews/control-system-hardware
      :type: ref
      :text: Go to Hardware Overview
      :classes: btn-outline-primary btn-block

   ---
   :header: font-weight-bold text-dark

   Software Overview

   ^^^

>    An overview of the software components and tools available for teams.

Reviewer 2: ```suggestion
   An overview of the software components and tools available to teams.
```

   +++

   .. link-button:: /docs/controls-overviews/control-system-software
      :type: ref
      :text: Go to Software Overview
      :classes: btn-outline-primary btn-block

.. raw:: html

   <hr>

.. panels::
   :header: text-black font-weight-bold
   :column: col-sm-4 d-flex pb-3
   :footer: bg-light

   VS Code Basics

   ^^^

>    VS Code is the official Integrated Development Environment for FRC\ |reg|. These documents highlight the usage necessary for teams.

Reviewer: ```suggestion
   Visual Studio Code is the official Integrated Development Environment for FRC\ |reg|. These documents highlight the usage necessary for teams.
```

   +++

   .. link-button:: /docs/software/vscode-overview/index
      :type: ref
      :text: Go to Page
      :classes: btn-outline-primary btn-block

   ---

   FRC Driver Station

   ^^^

>    FRC Driver Station provided by NI is the tool used for enabling and controlling your robot.

Reviewer 2: ```suggestion
   FRC Driver Station, provided by NI, is the tool used for enabling and controlling your robot.
```

   +++

   .. link-button:: /docs/software/driverstation/index
      :type: ref
      :text: Go to Page
      :classes: btn-outline-primary btn-block

   ---

   Hardware Basics

   ^^^

>    Get introduced into the FRC Control System Components and hardware for building your robot.

Reviewer 2: "Get introduced into" sounds awkward.

Reviewer: components shouldn't be capitalized

   +++

   .. link-button:: /docs/hardware/hardware-basics/index
      :type: ref
      :text: Go to Page
      :classes: btn-outline-primary btn-block

   ---

   roboRIO Information

   ^^^

>    roboRIO provided by NI is the robot processor responsible for directly controlling your robot.

Reviewer 2: ```suggestion
   The roboRIO, provided by NI, is the computer processor responsible for directly controlling your robot.
```

   +++

   .. link-button:: /docs/software/roborio-info/index
      :type: ref
      :text: Go to Page
      :classes: btn-outline-primary btn-block

   ---

   Command-Based

   ^^^

>    Command-Based programming is a programming framework that helps provide structure through subsytems and commands.

Reviewer 2: ```suggestion
   Command-Based programming is a framework that helps provide structure through subsytems and commands.
```

   +++

   .. link-button:: /docs/software/commandbased/index
      :type: ref
      :text: Go to Page
      :classes: btn-outline-primary btn-block

   ---

   Dashboards

   ^^^

>    Dashboards such as Shuffleboard and Glass are integral means of viewing diagnostic and other informative data from your robot.

Reviewer: ```suggestion
   Dashboards such as Shuffleboard and Glass are integral means of viewing diagnostic and other data from your robot.
```

   +++

   .. link-button:: /docs/software/dashboards/index
      :type: ref
      :text: Go to Page
      :classes: btn-outline-primary btn-block

> Not all content on the website is listed in the above panels. Please see the below sections for the full list of available content. Alternatively, you can use the search bar in the top-left to find content.

Reviewer: Do we need the TOC now, or can we just refer people to the TOC on the left?

Author: We can't easily hide the ToC from the body without messing up navigation.

Reviewer: It looks like there is a :hidden: directive that can be used to do this. https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#toctree-directive
